SwrveSDK/src/main/java/com/swrve/sdk/SwrveWakefulService.java

```diff
@@ -3,19 +3,20 @@
 import android.app.IntentService;
 import android.content.Intent;
 
-import com.swrve.sdk.localstorage.ILocalStorage;
 import com.swrve.sdk.localstorage.MemoryCachedLocalStorage;
+import com.swrve.sdk.localstorage.SQLiteLocalStorage;
+import com.swrve.sdk.rest.IRESTClient;
 import com.swrve.sdk.rest.RESTClient;
 
 import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Map;
 
 public class SwrveWakefulService extends IntentService {
 
     private static final String LOG_TAG = "SwrveWakeful";
     public static final String EXTRA_SEND_EVENTS = "swrve_wakeful_events";
 
+    private Swrve swrve = (Swrve) SwrveSDKBase.getInstance();
```

Comment:
If they haven't initialized the sdk, then SwrveWakefulService will  not have been called. This is only being used by the Locations sdk (so far).